src/configureWorkspace/configure.ts

```diff
@@ -57,8 +57,14 @@ export function getExposeStatements(ports: number[]): string {
     return ports ? ports.map(port => `EXPOSE ${port}`).join('\n') : '';
 }
 
-export function getComposePorts(ports: number[]): string {
-    return ports && ports.length > 0 ? '    ports:\n' + ports.map(port => `      - ${port}:${port}`).join('\n') : '';
+export function getComposePorts(ports: number[], debugPort?: number): string {
+    let portMappings: string[] = ports.map(port => `      - ${port}`);
```

Comment:
How about `ports?.map(port => `      - ${port}`)`? Since merging both in the ports array won't really work given we want the debugPort to be mapped to the same host port.